From f2acf100a3931ff03b7dbd3bf51971326ab83df5 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Fri, 9 Feb 2024 14:42:53 +0100
Subject: [PATCH 0942/1141] arm64: dts: st: update dsi node on stm32mp257f-ev1
 board

DSI needs 2 powers supplies VDD & VDDA18.

Change-Id: Ic8187392a17d4d2af6aafa8ce578975ccf4a8789
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/358853
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -206,7 +206,7 @@
 };
 
 &dsi {
-	phy-dsi-supply =  <&scmi_vddcore>;
+	vdd-supply =  <&scmi_vddcore>;
 	vdda18-supply = <&scmi_v1v8>;
 	status = "disabled";
 
